DOC-270 PR to update dynamo.md

james-macrometa opened the pull request:
We have made edits for clarity and to correct issues in the code samples. Ticket Reference: https://macrometa.atlassian.net/browse/DOC-270

james-macrometa committed:
DOC-270 PR to update dynamo.md

We have made edits for clarity and to correct issues in the code samples. Ticket Reference: https://macrometa.atlassian.net/browse/DOC-270

diana-macrometa commented on docs/tutorials/dynamo.md:
```diff
@@ -132,14 +103,31 @@ This flag can be set in `_guestDBs` collection for the corresponding geofabric.
 
 ### Scan
 
+**Retrieve All Items:**
```
I suggest changing the bold items in this section to h4s and removing the colons.

jamesm-macrometa replied:
Done

diana-macrometa reviewed (approved):
Approved with one minor suggestion.

jamesm-macrometa committed:
Merge branch 'master' into james-macrometa-patch-1

jamesm-macrometa committed:
Update dynamo.md

@james-macrometa Can you please take a look at the intro sentences I wrote for each major section and verify that they're accurate?

jamesm-macrometa commented:
Looks good, but we should wait for @james-macrometa to verify my additions before we merge.

jamesm-macrometa committed:
Update dynamo.md

james-macrometa commented:
Hi @jamesm-macrometa and @diana-macrometa I think everything looks good to me. Please proceed.
